src/DataWrangling/ECCO/ECCO.jl

```diff
@@ -2,6 +2,7 @@ module ECCO
 
 export ECCOMetadata, ECCO_field, ECCO_mask, ECCO_immersed_grid, adjusted_ECCO_tracers, initialize!
 export ECCO2Monthly, ECCO4Monthly, ECCO2Daily
+export ECCODarwinMetadata, ECCO4DarwinMonthly, ECCO270DarwinMonthly
```

Comment:
Just a note, "ECCO4" refers to ECCO version 4, but certainly "ECCO 270" does not refer to ECCO version 270. Do we have ideas for clearer naming?

Reply:
It's unclear to me, at least, if LLC270 will be ECCOv5, or if it's a stop-gap for a high res run, that's not CS510 (which is what you're calling ECCO2). Also, FYI there are 5 "releases" of ECCO version 4. ECCODarwin has results on V4r4 and V4r5, but I left that out for now, and just stuck to "ECCO4".

Reply:
Okay, well the key for the metadata is to define the grid and a way to set the variables. So if the different releases occupy the same pattern, then loading data from different releases can be described through properties of the types rather than the type names; for example `ECCO4Monthly(release=5)` or whatever.

We can call ECCO2 CS510 btw. As I understand it, "ECCO v2" refers to the run, whereas CS510 is the name of the grid "cubed sphere 510 points" or whatever. So the name "CS510" is less specific than "ECCO2"; there are many runs with CS510 that are not ECCO, or not on Earth at all. Similarly LLC270 is the name of the grid, right?